manager/integration/tests/backupstore.py

```diff
@@ -181,12 +207,19 @@ def backupstore_write_backup_cfg_file(client, core_api, volume_name, backup_name
                                     data)
 
     elif is_backupTarget_nfs(backupstore):
-        nfs_write_backup_cfg_file()
+        nfs_write_backup_cfg_file(client,
+                                  volume_name,
+                                  backup_name,
+                                  data)
 
 
-# TODO: implement nfs_write_backup_cfg_file
-def nfs_write_backup_cfg_file():
-    raise NotImplementedError
+def nfs_write_backup_cfg_file(client, volume_name, backup_name, data):
+    nfs_backup_cfg_file_path = nfs_get_backup_cfg_file_path(client,
+                                                            volume_name,
+                                                            backup_name)
+    cfg_file = open(nfs_backup_cfg_file_path, 'w')
+    cfg_file.write(json.dumps(data))
```

Comment:
In the minio case you use str(), I assume the data is a python object that should still be json encoded?
```
        tmp_bkp_cfg_file.write(str(backup_cfg_data))
```

Reply:
Fixed